docs/structured-streaming-programming-guide.md

```diff
@@ -530,6 +530,8 @@ Here are the details of all the sources in Spark.
         "s3://a/dataset.txt"<br/>
         "s3n://a/b/dataset.txt"<br/>
```

Comment:
I'd drop s3n  & s3 refs as they have gone from deprecated to deceased

Reply:
This looks like beyond of this PR: we can address it in separate PR. Could you raise another one?